src/typescript.gen.ts

```diff
@@ -134,7 +134,7 @@ export interface IDictionary<T> {
   }
 
   declareProperty(indent: string, property: IProperty) {
-    const optional = (property.nullable || !property.required) ? '?' : ''
+    const optional = !property.required ? '?' : ''
```

Comment:
Dang that was simple! Were there additional upstream changes?

Reply:
I'd already updated it in [this PR](https://github.com/looker-open-source/sdk-codegen/pull/35/files) and it looks like you incorporated that in [e5ebe98](https://github.com/looker-open-source/sdk-codegen/commit/e5ebe98afe38e9120e5d10022e13bd13fffa0261#diff-60c58a6f980980a1d10a67fe46d1169aR100) but you left in the `property.nullable` check